100DaysSmiling/Info.plist

```diff
@@ -2,6 +2,10 @@
 <!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
 <plist version="1.0">
 <dict>
+	<key>NSMicrophoneUsageDescription</key>
```

Comment:
It's mandatory to have the microphone if we are not using? Just about the privacy and that some people have some concerns about

Reply:
For now, we'll have to live with it